Formula/telnet.rb

```diff
@@ -25,21 +25,24 @@ def install
       ENV["SDKROOT"] = MacOS.sdk_path
       ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
 
-      # Force 64 bit-only build, otherwise it fails on Mojave
-      xcodebuild "SYMROOT=build", "-arch", "x86_64"
+      arch = Hardware::CPU.arm? ? "arm64" : "x86_64"
+
+      xcodebuild "SYMROOT=build", "-arch", arch
```

Comment:
Might be cleaner to try (untested):

```suggestion
      xcodebuild "SYMROOT=build", "-arch", Hardware::CPU.arch
```

(and similarly for `RC_ARCHS`)

Reply:
Good idea, will give this a test 👍